### PR TITLE
feat: add gio-table-light component

### PR DESCRIPTION
**Pull request by gcusnieux**
**Issue**

https://github.com/gravitee-io/issues/issues/6763

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://6183b02d73381a003a3be1a6-xrcwmrsaza.chromatic.com)

### CODE REVIEW

File: ui-particles-angular/projects/ui-particles-angular/src/scss/component/gio-table-light/gio-table-light.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+$background: map.get(theme.$mat-theme, background);
+
+@mixin theme() {
+  table.gio-table-light {
+    border-collapse: collapse;
+    border-spacing: 0;
+
+    th,
+    td {
+      // FIXME: add color to palette ? replace/keep ?
```
Comment (gaetanmaisse) [inline ui-particles-angular/projects/ui-particles-angular/src/scss/component/gio-table-light/gio-table-light.scss:32]: To handle now or it can wait?

**Review by gaetanmaisse — APPROVED** [review]

File: ui-particles-angular/projects/ui-particles-angular/src/scss/component/gio-table-light/gio-table-light.scss (hunk 1 above)
Comment (gcusnieux) [reply ui-particles-angular/projects/ui-particles-angular/src/scss/component/gio-table-light/gio-table-light.scss:32]: We can wait

Comment (gaetanmaisse) [reply ui-particles-angular/projects/ui-particles-angular/src/scss/component/gio-table-light/gio-table-light.scss:32]: 🚀

**Pull request merged**